README.md

```
@@ -10,6 +10,8 @@ An innovative educational platform that transforms business case studies into im
 
 ## 🚀 Quick Start
 
+my silly change 
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)